Sources/SwiftInspectorVisitors/ClassVisitor.swift

```diff
@@ -147,11 +147,3 @@ public final class ClassVisitor: SyntaxVisitor {
   private var classInfo: ClassInfo?
   private var innerClasses = [ClassInfo]()
 }
-
-public struct ClassInfo: Codable, Equatable {
```

Comment:
this type is moved to below with no changes (unless you are in file-changed view, at which point the comment is moved to a new type)